octobot_evaluators/evaluators/abstract_evaluator.py

```diff
@@ -232,7 +319,24 @@ async def stop(self) -> None:
         implement if necessary
         :return: None
         """
-        pass
+        await self.close_caches()
+        for consumer in self.consumers:
+            await consumer.stop()
+
+    async def clear_all_cache(self):
+        for evaluator_name in [self.get_name()] + [evaluator.get_name() for evaluator in
+                                                   self.called_nested_evaluators]:
+            await databases.CacheManager().clear_cache(evaluator_name)
```

Comment:
I'm not sure it's up to the Evaluator to know what the name of his cache is.

Reply:
Right, I'll move this to its own file